src/dotnet/Chat.UI.Blazor/Components/MarkupParts/CodeBlockMarkupView/code-block-markup-view.ts

```diff
@@ -47,3 +47,16 @@ export function highlightCode(pre: HTMLPreElement, languageName: string, code: s
         errorLog?.log(`highlightCode: failed to highlight code`, e);
     }
 }
+
+function applyTheme(themeInfo: ThemeInfo){
+    if (themeInfo.currentTheme === 'light') {
```

Comment:
Does importing theme css replaces the old one, right?

Reply:
on import

Reply:
sorry, was answering on previous question

Reply:
yes